freqtrade/rpc/rpc.py

```diff
@@ -849,16 +849,23 @@ def _rpc_reload_config(self) -> dict[str, str]:
         self._freqtrade.state = State.RELOAD_CONFIG
         return {"status": "Reloading config ..."}
 
-    def _rpc_stopentry(self) -> dict[str, str]:
+    def _rpc_pause(self) -> dict[str, str]:
         """
-        Handler to stop buying, but handle open trades gracefully.
+        Handler to pause trading (stop entering new trades), but handle open trades gracefully.
         """
         if self._freqtrade.state == State.RUNNING:
-            # Set 'max_open_trades' to 0
-            self._freqtrade.config["max_open_trades"] = 0
-            self._freqtrade.strategy.max_open_trades = 0
+            self._freqtrade.state = State.PAUSED
 
-        return {"status": "No more entries will occur from now. Run /reload_config to reset."}
+        if self._freqtrade.state == State.STOPPED:
+            self._freqtrade.state = State.PAUSED
+            return {
+                "status": "starting bot with trader in paused state, no entries will occur. \
```

Comment:
Don't split strings like htis - that's bad for 2 reasons
1 - splitting lines with `\` is very error prone
2 - it causes rendering issues

![image](https://github.com/user-attachments/assets/c4672e58-c6dd-4960-899c-b98ad896cdf0)